groovy/libs.gradle/nbproject/project.properties

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+is.autoload=true
+nbm.needs.restart=true
+
+javac.source=1.8
+javac.compilerargs=-Xlint -Xlint:-serial
+
+# For more information, please see http://wiki.netbeans.org/SignatureTest
+sigtest.gen.fail.on.error=false
```

Comment:
Wouldn't it make sense to run sigtest - this could detect incompatible changes and indicate when using modules need to be checked.

Reply:
Well, I stole that line from the libs.groovy module. As these Gradle Tooling update happens in a controlled way the dependent modules are checked as well.
What should be the procedure if I leave this one true and the sigtest fails? I would not like to bump the release-version of this module in that case.

Reply:
Why not bump the release version? The benefit I see is, that everybody is aware that a potentially problematic update happened.
The last time JNA was updated, it transitioned from 4.X to 5.Y and as this might break thing I bumped release version. All modules that depend on JNA had to be updated but that was not so much work - having all modules open checks for compatibility can directly applied.

Reply:
Between Gradle 6.0 and 6.3 Gradle changed some API-s which are in incubation. And according to their practices, they are free to do that. The stable API-s remains unchanged between major versions. Probably the release version can go by the major version, I'm going to modify this PR according to that, but I still think sigtests would be an unnecessary overhead in this process.

Reply:
Ok - I see. With that informatio it indeed makes no sense.